projects/github-actions/repo-gardening/changelog/rm-update-board-task

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: changed
+
+Board Triage: remove updateBoard task. It will now be part of the existing triageIssues task.
```

Comment:
I've been wondering if I should change this to a major bump. It doesn't create errors in the action if you continue to have the `updateBoard` task set in your workflow file, but that task will just not run, thus breaking your expectations.

Reply:
I've thought about it and switched it to a major bump in fecf1ec153f5a1b079ddb0fa90defb536370d529